Gemfile

```diff
@@ -257,7 +257,7 @@ if ENV["IMPORT"] == "1"
 
   # NOTE: in import mode the version of sqlite can matter a lot, so we stick it to a specific one
   gem 'sqlite3', '~> 1.3', '>= 1.3.13'
-  gem 'ruby-bbcode-to-md', git: 'https://github.com/nlalonde/ruby-bbcode-to-md'
+  gem 'ruby-bbcode-to-md', git: 'https://github.com/harry-wood/ruby-bbcode-to-md', branch: 'fluxbb-tests-and-fixes'
```

Comment:
Could you please create a PR with your changes at https://github.com/nlalonde/ruby-bbcode-to-md?

Reply:
Indeed I have done so (see description above). So I'm hoping this Gemfile change would eventually not be necessary, although... I don't know if nlalonde is intending to merge any PRs there and maintain that fork of the gem. D'you know what the status of that is?

Reply:
He's on our team, so I'm sure he will merge the PR when it's ready for review. Just mention me here when your PRs are ready for review and I'll get everything sorted.